docs/source/authentication.rst

```diff
@@ -80,258 +34,75 @@ listed below are configured under the ``auth`` section in the configuration file
-+----------------+--------------------------------------------------------+
-| debian package | /usr/lib/python2.7/dist-packages/st2auth               |
-+----------------+--------------------------------------------------------+
-| fedora package | /usr/lib/python2.7/site-packages/st2auth               |
-+----------------+--------------------------------------------------------+
-
-Add the following line to /etc/apache2/ports.conf. ::
-
-    Listen 9100
-
-Enable SSL and st2-auth and restart Apache. ::
-
-    sudo ln -s /etc/apache2/sites-available/st2-auth.conf /etc/apache2/sites-enabled/st2-auth.conf
-    sudo a2enmod ssl
-    sudo service apache2 restart
-
-Setup standalone mode
----------------------
-
-This section describes authentication backends which are available for you to use in the standalone
-mode.
-
-Keep in mind that only htpasswd compatible flat file authentication backend is available and
-installed by default.
-
-If you want to use a different backend (e.g. Keystone) you need to install a corresponding Python
-package for that backend and configure st2auth as described below.
-
-For example, to install keystone backend package, you would run the command shown below:
-
-.. sourcecode:: bash
-
-    pip install git+https://github.com/StackStorm/st2-auth-backend-keystone.git@master#egg=st2_auth_backend_keystone
-
-Keep in mind that you need to run this command on the same server where st2auth is running.
-
-Flat file backend
-~~~~~~~~~~~~~~~~~
-
-Flat file backend supports reading credentials from an Apache HTTPd htpasswd formatted file. To
-manage this file you can use `htpasswd`_ utility which comes with a standard Apache httpd
-distribution or by installing apache2-utils package on Ubuntu / Debian.
-
-**Backend configuration options:**
-
-* ``file_path`` - Path to the file containing credentials.
-
-Example htpasswd command to generate a password file with a user entry.
+The service can be configured with different backends (i.e. PAM, LDAP, etc.) to handle the
+authentication. If backend is not specified, a htpasswd compatible flat file authentication
+backend is used. The all-in-one installer and packages download and configure PAM by default. To
+use a different backend, select and install the appropriate python package from the |st2| 
+`community repos <https://github.com/StackStorm?utf8=✓&query=st2-auth>`_ and configure st2auth
+accordingly. For example, to install the package for the PAM backend manually, run the following
+command on the same server where st2auth is running.
 
 .. sourcecode:: bash
 
-    htpasswd -cs /path/to/.htpasswd stark
+    pip install git+https://github.com/StackStorm/st2-auth-backend-pam.git@master#egg=st2_auth_backend_pam
 
-Example ``auth`` section in the |st2| configuration file.
+After the backend is installed, configure the backend at /etc/st2/st2.conf. Specific configuration
+details for the backend can be found in the README at the corresponding repo. The following is a
```

Comment:
I assume you will move documentation to the corresponding repos? Or you have already done it?